src/components/Form/Number/Number.jsx

```diff
@@ -94,23 +91,23 @@ export default class Number extends ValidationElement {
 
     if (!isNaN(parseInt(this.state.value))) {
       if (status && this.props.min) {
-        status = status && parseInt(this.state.value) >= parseInt(this.props.min)
+        status = parseInt(this.state.value) >= parseInt(this.props.min)
         if (status === false) {
           errorCode = 'min'
         }
         hits++
       }
 
       if (status && this.props.max) {
-        status = status && parseInt(this.state.value) <= parseInt(this.props.max)
+        status = parseInt(this.state.value) <= parseInt(this.state.max)
         if (status === false) {
           errorCode = 'max'
         }
         hits++
       }
 
       if (status && this.props.maxlength && this.props.maxlength > 0) {
-        status = status && ('' + this.state.value).length <= parseInt(this.props.maxlength)
+        status = ('' + this.state.value).length <= parseInt(this.props.maxlength)
```

Comment:
what does prepending a string with an empty string do in javascript? do I even want to know?